consensus/src/network.rs

```diff
@@ -169,31 +169,31 @@ impl NetworkSender {
     }
 
     pub async fn broadcast_proposal(&mut self, proposal_msg: ProposalMsg) {
-        fail_point!("consensus::send_proposal", |_| ());
+        fail_point!("consensus::broadcast_proposal", |_| ());
         let msg = ConsensusMsg::ProposalMsg(Box::new(proposal_msg));
         self.broadcast(msg).await
     }
 
     pub async fn broadcast_sync_info(&mut self, sync_info_msg: SyncInfo) {
-        fail_point!("consensus::send_sync_info", |_| ());
+        fail_point!("consensus::broadcast_sync_info", |_| ());
         let msg = ConsensusMsg::SyncInfo(Box::new(sync_info_msg));
         self.broadcast(msg).await
     }
 
     pub async fn broadcast_timeout_vote(&mut self, timeout_vote_msg: VoteMsg) {
-        fail_point!("consensus::send_vote", |_| ());
+        fail_point!("consensus::broadcast_timeout_vote", |_| ());
```

Comment:
Everything else is just aesthetics. This is the only significant change (it conflicts with L210)